pubsub/api/Pubsub.Samples.Tests/CreateTopicWithAzureEventHubsIngestionTest.cs

```diff
@@ -31,20 +32,8 @@ public CreateTopicWithAzureEventHubsIngestionTest(PubsubFixture pubsubFixture)
     public void CreateTopicWithAzureEventHubsIngestion()
     {
         string topicId = _pubsubFixture.RandomTopicId();
-        var (resourceGroup, nameSpace, eventHub, clientId, tenantId, subscriptionId, gcpServiceAccount) = _pubsubFixture.RandomAzureEventHubsIngestionParams();
-        Topic createdTopic = _createTopicWithAzureEventHubsIngestionSample.CreateTopicWithAzureEventHubsIngestion(_pubsubFixture.ProjectId, topicId, resourceGroup, nameSpace, eventHub, clientId, tenantId, subscriptionId, gcpServiceAccount);
-
-        // Confirm that the created topic and topic retrieved by ID are equal
-        Topic retrievedTopic = _pubsubFixture.GetTopic(topicId);
-        Assert.Equal(createdTopic, retrievedTopic);
-
-        // Confirm that all Ingestion params are equal to expected values
-        Assert.Equal(resourceGroup, createdTopic.IngestionDataSourceSettings.AzureEventHubs.ResourceGroup);
-        Assert.Equal(nameSpace, createdTopic.IngestionDataSourceSettings.AzureEventHubs.Namespace);
-        Assert.Equal(eventHub, createdTopic.IngestionDataSourceSettings.AzureEventHubs.EventHub);
-        Assert.Equal(clientId, createdTopic.IngestionDataSourceSettings.AzureEventHubs.ClientId);
-        Assert.Equal(tenantId, createdTopic.IngestionDataSourceSettings.AzureEventHubs.TenantId);
-        Assert.Equal(subscriptionId, createdTopic.IngestionDataSourceSettings.AzureEventHubs.SubscriptionId);
-        Assert.Equal(gcpServiceAccount, createdTopic.IngestionDataSourceSettings.AzureEventHubs.GcpServiceAccount);
+        var (resourceGroup, nameSpace, eventHub, clientId, tenantId, subscriptionId, gcpServiceAccount) = _pubsubFixture.AzureEventHubsIngestionParams();
+        var exception = Assert.Throws<RpcException>(() =>  _createTopicWithAzureEventHubsIngestionSample.CreateTopicWithAzureEventHubsIngestion(_pubsubFixture.ProjectId, topicId, resourceGroup, nameSpace, eventHub, clientId, tenantId, subscriptionId, gcpServiceAccount));
+        Assert.Equal(StatusCode.NotFound, exception.Status.StatusCode);
```

Comment:
What's not found? If we can we should assert both on the status code and the error message, preferably with an explanation of why we are doing that.